.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,113 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches:
+      - master
+      - release
+      - 'v3.13-dev'
+  pull_request:
+    branches:
+      - master
+      - release
+      - 'v3.13-dev'
```

Comment:
Is there any reason to block CI from running if there was ever a pull request to another branch?
There might be other better-looking ways to do it, but `pull_request:` by itself with nothing following it enables the pull request trigger without putting filters on it. Maybe `branches: [ '*' ]` underneath if that looks too weird.

Reply:
You are probably right. I might change this in my next PR.

Reply:
I wonder if `pull_request` with no `:` would have worked.

Reply:
> if there was ever a pull request to another branch?

It sounds unusual, but I've contributed directly to the topic branches of another person's PRs on GitHub. PR inception. I've also contributed to a compiler feature branch which is not something NUnit currently does.